module #3. REST API Advanced/rest-advanced/service/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.0.6'
+	id 'io.spring.dependency-management' version '1.1.0'
+}
+
+group = 'com.epam.esm'
+version = '1.0'
+sourceCompatibility = '17'
+targetCompatibility = '17'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation project(':repository')
+	implementation project(':core')
+	implementation 'org.springframework.boot:spring-boot-starter-hateoas'
+	implementation ('org.springframework.boot:spring-boot-starter-data-jdbc') {
+		exclude group: 'org.springframework.boot', module: 'spring-boot-starter-logging'
```

Comment:
why `exclude group: 'org.springframework.boot'` is excluded?

Reply:
group: 'org.springframework.boot', module: 'spring-boot-starter-logging' excluded because I use Log4j2 logger implementation. So it requires spring boot standard logger disabling.